### DISCUSSION TITLE
Lock icon is too big

### DISCUSSION THREAD

**Discussion by pablomedok** [Sequel-Ace/Sequel-Ace#1390] [Ideas]
Thanks for adding a lock icon indicating SSL encryption. However it's too big and looks like a draft. Make it more subtle:

Before:
![Image 2022-02-14 at 5 03 49 PM](https://user-images.githubusercontent.com/13901628/153879849-d99f9389-58ef-40ee-9e1f-70c122b19b15.jpg)

After:
![Image-2022-02-14-at-5 03 49-PM](https://user-images.githubusercontent.com/13901628/153879988-ca983ccc-f34b-4603-95b7-ac53451ea390.png)

**Comment by Jason-Morcos** [comment]
PRs are welcome :)

**Comment by Kaspik** [comment]
What macOS version are you running @pablomedok ?
